devnet/src/types.rs

```diff
@@ -60,6 +61,44 @@ pub struct MpcNetworkSetup {
     pub nomad_server_url: Option<String>,
 }
 
+impl fmt::Display for MpcNetworkSetup {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
For consistency since you have `use std::fmt`.
```suggestion
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> fmt::Result {
```